webdriver/tests/bidi/script/add_preload_script/contexts.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("domain", ["", "alt"], ids=["same_origin", "cross_origin"])
+async def test_iframe(
```

Comment:
So what about the following? I think that this fits indeed better.

```suggestion
async def test_top_context_with_iframes(
```